index.bs

```diff
@@ -841,6 +841,31 @@ that authors can use to extend the language
 without breaking future native functionality,
 to reduce such dilemmas in the future.
 
+<h3 id="text-based-syntaxes">Design text-based syntax with humans in mind</h3>
+
+When designing a text-based syntax, consider human usability and readability,
+even when you expect the syntax to be primarily generated by tooling.
+
+Historically, every syntax that could be read by humans, inevitably ended up being authored by humans too.
+Even when syntax is primarily authored by tools,
+it is still effectively authored by humans: the tool developers.
+Therefore, optimizing it for human usability also makes it easier to develop tools to generate it.
+
+Human usability may often be at odds with parsing performance and filesize efficiency.
+Ideally, the syntax would be optimized for human usability, and tooling would optimize the syntax for efficiency.
+However, this is not always possible.
+In these cases, explore if it is possible to provide syntax that is flexible enough to accommodate both use cases.
+The downside of this approach is that it increases the complexity of the language, and humans may still need to read the less human-friendly syntax.
+Lastly, if efficiency is truly critical, consider whether a binary format may be more appropriate.
+
+<div class="example">
```

Comment:
@plinss and @LeaVerou also gave JSON as a great example (+1 to both of these from me):

* Not allowing trailing commas is not good for humans.
* Not allowing comments is *really* not good for humans.